html/utilities/validation/setValidTick.js

```diff
@@ -1,4 +1,4 @@
-function setValidTick(inputContainer) {
+const setValidTick = (inputContainer) => {
```

Comment:
hey @drishit96  thanks for contributing! Could you add a `;` to the end of this arrow function please? Thanks again!

Reply:
Done